src/getting_started/parsec_tool_use.md

```diff
@@ -0,0 +1,536 @@
+to hear from users with your questions and queries. Head over to the
+[community](https://github.com/parallaxsecond/community) repo to learn how to get in touch via Slack
+and Zoom.
+
+## Checking the Service Back-End Configuration
+
+The `parsec-tool` includes commands that can help you check on how the service has been set up.
+These checks can be useful, for example, if you wish to make sure that the Parsec service is using
+the correct secure hardware on your device, such as your Trusted Platform Module (TPM) or Hardware
+Security Module (HSM).
+
+To check the hardware back-end configuration of Parsec on your device, run this command:
+
+```
+parsec-tool list-providers
+```
+
+This command displays a list of the back-end provider modules that have been activated in the Parsec
+service. The entry at the **top** of this list is the most important one, because this entry tells
+you which back-end provider will be used by default. You should pay particular attention to this
+entry, and ensure that it matches your expectations for how Parsec has been integrated with your
+device hardware.
+
+If you are experimenting or prototyping with Parsec in a Proof-of-Concept (PoC) or non-production
+environment, you will probably be using the software back-end (also known as the *Mbed Crypto* or
+*Mbed TLS* back-end). This is the back-end provider that is configured out of the box in new Parsec
+installations. It will appear in the list as follows:
+
+```
+ID: 0x01 (Mbed Crypto provider)
+Description: User space software provider, based on Mbed Crypto - the reference implementation of the PSA crypto API
+Version: 0.1.0
+Vendor: Arm
+UUID: 1c1139dc-ad7c-47dc-ad6b-db6fdb466552
+```
+
+If you are using this back-end, then you should be aware that it does not offer any specific
+security features. This back-end provider is suitable for evaluation and experimentation only.
+
+For Parsec to be integrated with a hardware or firmware TPM on your device, the topmost back-end
+provider should be the TPM provider, which will appear as follows:
+
+```
+ID: 0x03 (TPM provider)
+Description: TPM provider, interfacing with a library implementing the TCG TSS 2.0 Enhanced System API specification.
+Version: 0.1.0
+Vendor: Trusted Computing Group (TCG)
+UUID: 1e4954a4-ff21-46d3-ab0c-661eeb667e1d
+```
+
+For Parsec to be integrated with a hardware module using the PKCS#11 interface, the topmost entry
+should be:
+
+```
+ID: 0x02 (PKCS #11 provider)
+Description: PKCS #11 provider, interfacing with a PKCS #11 library.
+Version: 0.1.0
+Vendor: OASIS Standard.
+UUID: 30e39502-eba6-4d60-a4af-c518b7f5e38f
+```
+
+For hardware accessed through a `CryptoAuthLib` interface, the entry should be:
+
+```
+ID: 0x05 (CryptoAuthentication Library provider)
+Description: User space hardware provider, utilizing MicrochipTech CryptoAuthentication Library for ATECCx08 chips
+Version: 0.1.0
+Vendor: Arm
+UUID: b8ba81e2-e9f7-4bdd-b096-a29d0019960c
+```
+
+For Parsec to integrate with a [PSA
```

Comment:
Format: Sudden breaklines

Reply:
Again, I think this is just `vmdfmt` artefacts, but the CI will fail if we structure it differently. The important thing is whether it renders correctly.